WordPress/src/androidTest/java/org/wordpress/android/ui/screenshots/WPScreenshotTest.java

```diff
@@ -59,65 +55,112 @@ public void wPScreenshotTest() {
         Screengrab.setDefaultScreenshotStrategy(new UiAutomatorScreenshotStrategy());
 
         wpLogin();
+
+        // Never show the Gutenberg dialog when opening a post
+        AppPrefs.setGutenbergInformativeDialogDisabled(true);
+
+        tmpWPLogin();
         editBlogPost();
-        navigateReader();
-        navigateNotifications();
+        manageMedia();
         navigateStats();
-        wpLogout();
+        navigateNotifications();
+        tmpWpLogout();
     }
 
-    private void navigateReader() {
-        // Choose the "Reader" tab in the nav
-        clickOn(R.id.nav_reader);
+    private void tmpWPLogin() {
+        // If we're already logged in, log out before starting
+        if (!isElementDisplayed(R.id.login_button)) {
+            this.__wpLogout();
```

Comment:
I'm getting a compile error here, did you mean

```suggestion
            this.wpLogout();
```

Reply:
Ah – good catch! My bad.

That's fixed now, but the reason I brought in the old login / logout methods was because the new ones were failing for screenshots – in particular in other languages (Thai and Japanese seemed especially problematic). I spent some time with it and couldn't sort out why. I figured that could be something for a future PR? 

WDYT?

Reply:
Oh I meant to ask that, I'm not sure why they'd be failing, I actually didn't test on other languages so I'm not quite aware about what's happening, I'll look into it though thanks for pointing that out 👍🏽